source/docs/software/kinematics-and-odometry/swerve-drive-odometry.rst

```diff
@@ -63,35 +74,33 @@ This ``update`` method must be called periodically, preferably in the ``periodic
 
       @Override
       public void periodic() {
-        // Get my gyro angle. We are negating the value because gyros return positive
-        // values as the robot turns clockwise. This is not standard convention that is
-        // used by the WPILib classes.
-        var gyroAngle = Rotation2d.fromDegrees(-m_gyro.getAngle());
+        // Get my gyro angle.
+        var gyroAngle = m_gyro.getRotation2d();
 
         // Update the pose
-        m_pose = m_odometry.update(gyroAngle, m_frontLeftModule.getState(), m_frontRightModule.getState(),
-            m_backLeftModule.getState(), m_backRightModule.getState());
+        m_pose = m_odometry.update(gyroAngle,
+          m_frontLeftModule.getPosition(), m_frontRightModule.getPosition(),
+          m_backLeftModule.getPosition(), m_backRightModule.getPosition());
```

Comment:
Following discord discussion - we're likely getting rid of variadic function signatures, so the docs should recommend array construction.

```suggestion
          new SwerveModulePosition[]{
            m_frontLeftModule.getPosition(),
            m_frontRightModule.getPosition(),
            m_backLeftModule.getPosition(),
            m_backRightModule.getPosition()
          });
```